lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -9,20 +10,27 @@ accounting:
   default_value: true
 
 acct-port:
-  kind: int
-  get_command: "show running-config radius all"
+  multiple: true
   get_value: '/^radius-server host %s.* acct-port (\d+).*$/'
   set_value: '<state> radius-server host <ip> acct-port <port>'
   default_value: 1813
+  nexus:
+    get_command: "show running-config radius all"
+  ios_xr:
+    get_command: "show running-config radius-server"
```

Comment:
Consider creating a `_template` section in this YAML so you don't have to repeat the `get_command` definition over and over.